templates/deployment.yaml

```diff
@@ -541,9 +545,31 @@ spec:
           mountPath: /opt/bird/
         - name: bird-socket
           mountPath: /var/run/bird/
+      {{- if .Values.bird.birdConfigWatcher.enabled }}
+      - name: bird-config-watcher
+        image: {{ .Values.bird.birdConfigWatcher.image.repository }}:{{ .Values.bird.birdConfigWatcher.image.tag }}
+        imagePullPolicy: {{ .Values.bird.birdConfigWatcher.image.pullPolicy | quote }}
+        resources:
+{{ toYaml .Values.resources.birdConfigWatcher | indent 10 }}
+        securityContext:
+          capabilities:
+            add:
+            - NET_ADMIN
+            - SYS_ADMIN
+            - SETPCAP
+            - NET_RAW
+        ports:
+        - name: p8000
+          containerPort: 8000
```

Comment:
Hm... this port shouldn't be hardcoded in the `bird-config-watcher`, it should be possible to configure it by env variable or flag.

Reply:
The issue is it is hardcoded in the code, so changing only in the chart will break the functionality

Reply:
It was changed in the linked commit (https://github.com/openvnf/bird-configwatcher/commit/60c66388e90490123aa3e187ca89979bc32306cb)

Reply:
Ok, see, also updated in bird config watcher